debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -5,22 +5,6 @@
  */
 package io.debezium.connector.mysql;
 
-import java.io.IOException;
```

Comment:
Would it be possible to keep the import order as is, i.e. the `java.*` types before any others? Otherwise we'll have lots of changes like this going forth and back, which makes it hard to focus on what has actually changed.

Reply:
Sure. would do that. But typically the style-checks doesn't allow the * types import to go through.